hack/.golint_failures

```diff
@@ -343,6 +343,7 @@ pkg/ssh
 pkg/util/bandwidth
 pkg/util/config
 pkg/util/configz
+pkg/util/conntrack
```

Comment:
Please fix the lint warnings by adding comments to the exported names instead of adding this package to the ignore file. I am worried that this package will stay here for ever if we don't fix this now and the fix is not that involved. See the lint failures and suggestions here: https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/59286/pull-kubernetes-verify/75380/#verify-golint